dev-python/ipywidgets/ipywidgets-9999.ebuild

```diff
@@ -29,6 +29,7 @@ RDEPEND="
 	"
 DEPEND="${RDEPEND}
 	test? (
+		$(python_gen_cond_dep 'dev-python/mock[${PYTHON_USEDEP}]' python2_7)
```

Comment:
@jlec This is similar to https://github.com/gentoo/gentoo/blob/master/dev-python/ipython/ipython-3.2.1.ebuild#L67 and I assume mock is really only needed for python 2.7 (https://github.com/ipython/ipywidgets/blob/master/setup.py#L286).